bundle/manifests/splunk-operator.clusterserviceversion.yaml

```diff
@@ -799,14 +802,14 @@ spec:
                     fieldRef:
                       fieldPath: metadata.annotations['olm.targetNamespaces']
                 - name: RELATED_IMAGE_SPLUNK_ENTERPRISE
-                  value: splunk/splunk:9.0.1-a1
+                  value: docker.io/splunk/splunk:edge
                 - name: OPERATOR_NAME
                   value: splunk-operator
                 - name: POD_NAME
                   valueFrom:
                     fieldRef:
                       fieldPath: metadata.name
-                image: splunk/splunk-operator:2.1.0
+                image: controller:latest
```

Comment:
need to check with @vivekr-splunk if this is correct.

Reply:
you can leave this as it is , i need to generate bundle file. once you merge the code, i will create the new PR just for bundle generation.